### ISSUE TITLE
Django 1.9 support

### ISSUE THREAD

**Issue by scott-w** [ambitioninc/django-query-builder#48]
Query Builder breaks in Django 1.9:

```
    from querybuilder.query import Query, SumField
  File "/local/lib/python2.7/site-packages/querybuilder/query.py", line 4, in <module>
    from django.db.models import Q, get_model
ImportError: cannot import name get_model
```

**Comment by scott-w** [comment]
It seems the fix is documented here: https://docs.djangoproject.com/en/1.9/ref/applications/#methods

**Comment by wesokes** [comment]
@scott-w Thanks for fixing this issue! Unfortunately, it looks like django 1.9 also changes some other paths that cause issues for a required package. I have a fix almost ready, and I'll let you know as soon as it is published.

**Comment by wesokes** [comment]
fixed in version 0.10.0
